scripts/build/Jenkins/Jenkinsfile

```diff
@@ -17,15 +19,61 @@ PIPELINE_TIMEOUT = 60
 
 EMPTY_JSON = readJSON text: '{}'
 
+ENGINE_DEFAULT_ORGANIZATION_NAME = 'o3de'
```

Comment:
Yes, but since this is for AR, AR should only be considered passing if the o3de is used right?

Reply:
It's possible to setup parameters so that they cannot be changed during AR. For example we allow enabling/disabling platforms for branch builds but not for AR. I'm fine with leaving this as is since it may take some additional testing to add it. 

I recommended we follow up with a task to investigate setting this as an optional parameter.

Reply:
ok